### PR TITLE
Use HTTPS for Minetest's website

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Use HTTPS for Minetest's website (in the header).

### CODE REVIEW

File: template/overall_header.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
 <!--
 	phpBB style name: prosilver
 	Based on style:   prosilver (this is the default phpBB3 style)
-	Original author:  Tom Beddard ( http://www.subBlue.com/ )
+	Original author:  Tom Beddard ( }://www.subBlue.com/ )
```
Comment (x2048) [inline template/overall_header.html:29]: Isn't this a typo?

Comment (Panquesito7) [reply template/overall_header.html:29]: Yeah, didn't notice I did that. Will fix. Thanks for noticing! 👍

**Review by rubenwardy — APPROVED** [review]

**Comment by sfan5** [comment]
The master branch is no longer in use, no point in making changes.

**Comment by Panquesito7** [comment]
> The master branch is no longer in use, no point in making changes.

Oh, so you're using another source for the forum now? 🤔

**Pull request closed without merging**